docs/package_templates/header_only/all/conanfile.py

```diff
@@ -43,18 +42,16 @@ def _compilers_minimum_version(self):
         }
 
     # Use the export_sources(self) method instead of the exports_sources attribute.
-    # This allows finer grain exportation of patches per version
     def export_sources(self):
         export_conandata_patches(self)
 
     def layout(self):
-        # src_folder must use the same source folder name than the project
         basic_layout(self, src_folder="src")
 
     def requirements(self):
         # Prefer self.requires method instead of requires attribute
         # Direct dependencies of header only libs are always transitive since they are included in public headers
-        self.requires("dependency/0.8.1", transitive_headers=True)
```

Comment:
`transitive_headers=True` is already the default for `package_type = "header-library"`.